utility/rp_push_client.py

```diff
@@ -0,0 +1,188 @@
+import argparse
+import os
+from concurrent.futures import ThreadPoolExecutor
+from functools import partial
+
+import xmltodict
+from rp_utils.preproc import PreProcClient
+from rp_utils.rp_client import Launch, ReportPortal, RpLog
+from rp_utils.xunit_xml import TestCase, TestSuite, XunitXML
+
+from utility.log import Log
+
+log = Log(__name__)
+doc = """
+Standard script to collect all the logs from ceph cluster
+
+    Usage:
+        rp_push_client.py --config_file <str> --payload_dir <str>
+        rp_push_client.py (-h | --help)
+
+    Options:
+        -h --help          Shows the command usage
+        -c --config_file <str>         Config file with report portal details.
+        -d --payload_dir <str>   Paylod directory where we have results and attachments
+"""
+
+
+def upload_logs_RP(args):
```

Comment:
This being the entry point. we could add more information about the workflow.

I believe it is sufficient to have the method name as `upload_logs`